e2e/test/scenarios/embedding/sdk-iframe-embedding/embed-options.cy.spec.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  FIRST_COLLECTION_ID,
+  ORDERS_DASHBOARD_ID,
+  ORDERS_QUESTION_ID,
+} from "e2e/support/cypress_sample_instance_data";
+
+const { H } = cy;
+
+describe("scenarios > embedding > sdk iframe embed options passthrough", () => {
```

Comment:
The only goal of these tests are to see whether the embed options "passed through" to the SDK. We don't aim to test any edge cases other than the fact that it's passed to the underlying SDK components, so don't expect full permutation coverage here.